internal/mode/static/manager.go

```diff
@@ -210,6 +210,8 @@ func StartManager(cfg config.Config) error {
 			ngxPlusClient,
 			ngxruntimeCollector,
 			cfg.Logger.WithName("nginxRuntimeManager"),
+			nil,
+			nil,
```

Comment:
These need to be initialized for the real thing.

Reply:
@sjberman  btw to update this properly I think I need for ProcessHandler interface, add ProcessHandlerImpl, if this is even good approach, 

thank you !

Reply:
@sjberman will take a closer look to make this good